types/address.go

```diff
@@ -31,15 +31,15 @@ const (
 
 	// AddrLen defines a valid address length
 	AddrLen = 20
-	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
+	// Bech32MainPrefix defines the main SDK Bech32 prefix of an account's address
 	Bech32MainPrefix = "cosmos"
 
-	// Atom in https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+	// CoinType is the ATOM coin type as defined in SLIP44 (https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
 	CoinType = 118
 
-	// BIP44Prefix is the parts of the BIP44 HD path that are fixed by
-	// what we used during the fundraiser.
-	FullFundraiserPath = "44'/118'/0'/0/0"
+	// FullFundraiserPath is the parts of the BIP44 HD path that are fixed by
+	// what we used during the ATOM fundraiser.
+	FullFundraiserPath = "m/44'/118'/0'/0/0"
```

Comment:
not sure if this breaks anything cc: @ebuchman @zmanian?

Reply:
this would break compatibility with fundraiser user recovery I am pretty sure

Reply:
kk I'll write more tests

Reply:
I'm not sure tho. Needs a bit more detective work.

Reply:
Nevermind. I think this is fine and won't change the fundraiser keys generated from mnemonic.

it's backwards compatible.

Reply:
added tests and it doesn't break anything 👍

Reply:
What was Metamask doing with these paths? Was it throwing an error or actually returning a different result? I guess the `m` should be fully ignored besides validating that it's there. Does anyone know why the standard calls for it in the first place or what it stands for? Maybe `master`?